source/support/cluster/cluster.sqf

```diff
@@ -1,23 +1,28 @@
+/*
+    File: cluster.sqf
 
-// R_60mm_HE / R_80mm_HE /Bo_Air_LGB(no sounds)/ Grenade / Bo_Mk82
-//
-// _null = [1, 250, 1, 90, 1, 5, "grenade"] execVM "support\cluster\mapclickcluster.sqf"
-//
+    Author: Kibot
 
-_position = _this select 0;
-_salvos = _this select 1;
-_radius = _this select 2;
-_interval = _this select 3;
-_rps = _this select 4;
-_supportype = _this select 5; // 1 = arty, 2 = mortar, 3 = jdam
-_cost = _this select 6;
-_ammotype = _this select 7;
+    Description: 
+        Deploy a cluster of munitions at the specified location.
 
+    Parameter(s):
+        _this select 0 - POSITION - Center position of 'Strike Area'
```

Comment:
Why do you use `_this select 0` to denote the first parameter if you don't use `_this select` anyway.
You should maybe use something like `1:` to denote the order of the parameters.